Makefile

```diff
@@ -3,18 +3,18 @@
 #See clock.c for the license detail.
 
 SRC = ttyclock.c
-CC ?= gcc
+CC += gcc
```

Comment:
this is wrong - we don't want to add to the compiler! if someone does `make CC=cc` you will end up with `cc gcc` as a compiler...